integration_testing/features/super-admin/commands/super-admin-automatic-device-provisioning.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Super admin can provision a device in the same way the setup wizard would
+    Super admin needs to be able to customize the Kolibri settings by modifying the options.ini file and pointing to a JSON file in Paths -> AUTOMATIC_PROVISION_FILE
+
+  Background:
+    Given that the Kolibri server is not running
+      And I have opened the "options.ini" file which is located inside the ".kolibri" folder
+      And there are the following sections: [Cache], [Database], [Server], [Paths], [Urls], [Deployment], [Python], [Tasks], [Learn]
+      And I have created a JSON file with the following format: {
+        "facility_name": "My Facility",
+        "preset": "formal",
+        "facility_settings": {
+            "learner_can_edit_username": true,
+            "learner_can_edit_name": true,
+            "learner_can_edit_password": true,
+            "learner_can_sign_up": true,
+            "learner_can_delete_account": true,
+            "learner_can_login_with_no_password": false,
+            "show_download_button_in_learn": true
+        },
+        "device_settings": {
+            "language_id": "en",
+            "landing_page": "learn",
+            "allow_guest_access": true,
+            "allow_peer_unlisted_channel_import": true,
+            "allow_learner_unassigned_resource_access": true,
+            "name": "My Device",
+            "allow_other_browsers_to_connect": true
+        },
+        "superuser": {
+            "username": "superuser",
+            "password": "password"
+        }
+    }
+
+  Scenario: Setup a device by modifying the AUTOMATIC_PROVISION_FILE setting in the options.ini file
+    When I add "AUTOMATIC_PROVISION_FILE = <file path>" under the [Paths] section
+      And I save my changes
+      And I run the "kolibri start" command in the terminal or command prompt
```

Comment:
Does it display the output even without running the ``kolibri start`` command with the flag ``--foreground``?

Reply:
It probably won't, but the output would be in the logs, at least.

Reply:
I suspected as much, thank you!